logfile

```diff
@@ -0,0 +1,6 @@
+FATAL:  "/usr/local/var/postgres" is not a valid data directory
```

Comment:
Looks like a stray file got in — can you remove this?

Reply:
Yes I will